.gitignore

```
@@ -24,6 +24,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+_version.py
```

Forgot to change this in #175 I think.

```suggestion
_icepyx_version.py
```

---

See #180